definitions/ext-argocd/dashboard.json

```diff
@@ -0,0 +1,459 @@
+{
+  "name": "ArgoCD",
+  "description": null,
+  "pages": [
+    {
+      "name": "Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 2,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![ArgoCD Logo](https://avatars1.githubusercontent.com/u/30269780?s=110&v=4)\n\n# ArgoCD\n\n\nFilter dashboard by `clusterName='<cluster_name>'`."
+          }
+        },
+        {
+          "title": "Applications",
+          "layout": {
+            "column": 3,
+            "row": 1,
+            "width": 2,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId" : 0,
+                "query": "SELECT count('apps') as 'Apps' FROM (SELECT latest(argocd_app_info) AS 'apps' FROM Metric FACET name, clusterName LIMIT MAX)"
```

Comment:
I'm not fully sure this will work as intended. We append a `WHERE entity.guid = 'guid'` in here

@danipl can you double check this one before merging?

Reply:
As I understand it, the internal subquery query only returns `apps`, so the filter would not work as entity.guid attribute is not exposed from the response of the subquery. @shawnj did you verify with sample data?

Reply:
Let me know if this won't work and I can try to find a different way to represent the query.